### DISCUSSION TITLE
From AWS S3 Blob Storage to STRATO S3

### DISCUSSION THREAD

**Discussion by Mobidat** [microstream-one/microstream#448] [Q&A] [answered]
I'm an S3 Newbie. Does anyone know, how to adapt the S3 AWS Target Params to STRATO HiDrive S3 Params?

[AWS Doc MicroStream](https://docs.microstream.one/manual/storage/storage-targets/blob-stores/aws-s3.html)
[S3 HiDrive API Reference](https://developer.hidrive.com/hidrive-s3-api-reference/)

HiDrive don't support a JAVA S3-Client as AWS do. Perhaps there is a ready to start solution in the web.

**Comment by hg-ms** [comment, marked as answer]
Did you try to use our AWS S3 connector?

At first glance, HiDrive S3 claims to be compatible to the Amazon SDKs.
(https://www.strato.de/cloud-speicher/hidrive-s3/#collapse_4_nr2)

But we never tested that and don’t provide support for HiDrive S3 at the moment.
